website/content/en/docs/cloud-providers/AWS/launch-templates.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "Launch Templates and Custom Images"
+linkTitle: "Launch Templates"
+weight: 80
+---
+
+By default, Karpenter generates launch templates that use [EKS Optimized AMI](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-ami.html) for nodes. Often, users need to customize the node image to integrate with existing infrastructure or meet compliance requirements. Karpenter supports custom node images through Launch Templates. If you need to customize the node, then you need a custom launch template. 
+
+Note: By customizing the image, you are taking responsibility for maintianing the image, including security updates. In the default configuration, Karpenter will use the latest version of the EKS optimized AMI, which is maintained by AWS. 
+
+
+## Introduction
+
+Karpenter follows existing AWS patterns for customizing the base image of
+instances. More specifically, Karpenter uses [EC2 launch templates](https://docs.aws.amazon.com/autoscaling/ec2/userguide/LaunchTemplates.html). Launch
+templates may specify many values. The pivotal value is the base image (AMI).
+Launch templates further specify many different parameters related to networking, authorization, instance type, and more. 
+
+**Karpenter only implementes a subset of launch template fields, and some fields should not be set.**  
+
+This guide describes requirements for using launch templates with Karpenter, and later an example procedure.
+
+## Launch Template Requirements
+
+The Launch Template resource includes a large number of fields. AWS accepts launch templates with any subset of these fields defined.
+
+Certain fields are obviously critical, such as AMI and User Data. Some fields are useful for particular workloads, such as storage and IAM Instance Profile. 
+
+Finally, **the majority of Launch Template fields should not be set** (or will have no effect), such as network interfaces and instance type. 
+
+## Important Fields
+
+When creating a custom launch template, the AMI and User Data are the defining characteristics. Instance Profile (IAM Role) and Security Group (firewall rules) are also important for Karpenter. 
+
+### AMI
+
+AMI (Amazon Machine Image), is the base image/VM for a launch template.

```

Comment:
For those new to AWS, is it worth noting that the AMI is regional? (for that matter, I wonder if we should note that LTs are regional?)

Reply:
this is a good point.